### ISSUE TITLE
RFE: drop using `six` module

### ISSUE THREAD

**Issue by kloczek** [testing-cabal/fixtures#61]
https://adamj.eu/tech/2019/03/13/dropping-python-2-support/

Looking on the code
```console
[tkloczko@devel-g2v fixtures-3.0.0]$ grep -rw six
fixtures/_fixtures/logger.py:import six
fixtures/_fixtures/logger.py:        six.reraise(*sys.exc_info())
fixtures/fixture.py:import six
fixtures/fixture.py:                six.reraise(*err)
requirements.txt:six
```

**Comment by cjwatson** [comment]
This was already removed in c4dfe1c8e1.

**Comment by kloczek** [comment]
That commit does not apply celanly on top of last release 😞 
Do you have maybe some plans to make new release?
Lokking on https://github.com/testing-cabal/fixtures/compare/3.0.0...master .. last one was in 2016 😄

**Comment by cjwatson** [comment]
I was looking at that today.  I think it'd be nice to get #62 sorted out, but after that I should be able to organize a release.

**Comment by kloczek** [comment]
OK. I have registered that ticket in my BTS so I'll try to confirm that everything is fine after release.
Thx.

**Comment by cjwatson** [comment]
I've released fixtures 4.0.0.

**Comment by kloczek** [comment]
Thank you.
Just tested new version and so far all looks good 😄
